Ginger/Ginger/BusinessFlowWindows/ucBusinessFlowPage.xaml

```diff
@@ -0,0 +1,26 @@
+<UserControl x:Class="Ginger.BusinessFlowWindows.ucBusinessFlowPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"          
+      xmlns:usercontrols="clr-namespace:Amdocs.Ginger.UserControls" 
+      mc:Ignorable="d" 
+      d:DesignHeight="30" d:DesignWidth="365">
+
+    <Grid x:Name="xBFGrid" Background="{StaticResource $BackgroundColor_LightGray}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="30*"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="80"/>
+            <ColumnDefinition Width="235"/>
+            <ColumnDefinition Width="28"/>
+            <ColumnDefinition Width="*"/>
```

Comment:
you should use Grid Columns/Rows only when really needed because it makes the page design much more complex, here instead you only need to use DockPanel/StackPanel to achieve the same that all will be horizntal- ping me if need help

Reply:
Done